src/auth/local/local.strategy.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable, UnauthorizedException } from "@nestjs/common";
+import { PassportStrategy } from "@nestjs/passport";
+import { Strategy } from "passport-local";
+
+import { AuthService } from "../auth.service";
+import { type JwtUser } from "../jwt/jwt.interface";
+
+@Injectable()
+export class LocalStrategy extends PassportStrategy(Strategy) {
+  constructor(private readonly authService: AuthService) {
+    super({ usernameField: "account" });
+  }
+
+  async validate(account: string, password: string) {
+    const user = await this.authService.validateUser(account, password);
+
+    if (!user) {
+      throw new UnauthorizedException();
+    }
+    const payload: JwtUser = {
+      email: user.email,
```

Comment:
is this information going to pack into JWT token? The JWT token is not encrypted, contains any confidential information in payload is a serious security issue.